### ISSUE TITLE
Random Questions Thread

### ISSUE THREAD

**Issue by massma** [deepclimatenyc/seminar#6]
Hi All,

Quick question: I vaguely remember a discussion last week about a recent student who did their dissertation on dimensional reduction of GCMs down to very few dimesnions.  Does anyone remember/know the students' name or some of their papers?  Or, am I losing my mind and that never happened?

Thanks!

**Comment by sylviasullivan** [comment]
I wrote down the name Nandini Ramesh from our discussion: https://scholar.google.com/citations?user=zJVePl8AAAAJ&hl=en
But Ryan can correct me if that's wrong.

**Comment by rabernat** [comment]
Correct. Her paper on using empirical dynamic modeling in the context of ENSO variability is currently in review.
